plugins/se/CMakeLists.txt

```diff
@@ -3,4 +3,10 @@
 # that can be found in the LICENSE file at the root of the
 # Mumble source tree or at <https://www.mumble.info/LICENSE>.
 
-add_library(se SHARED "se.cpp")
+add_library(se SHARED
+	"se.cpp"
+
+	"../Process.cpp"
+	"../ProcessLinux.cpp"
+	"../ProcessWindows.cpp"
```

Comment:
I think you must not use `ProcessLinux` on Windows platforms. This seems to create an error on our CI